sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerConfig.java

```diff
@@ -386,6 +396,15 @@ public String getDcosSpace() {
     return Optional.ofNullable(envStore.getOptional(SERVICE_REGION_ENV, null));
   }
 
+  public Optional<String> getServiceNamespace() {
```

Comment:
This is duplicated in https://github.com/mesosphere/dcos-commons/pull/3102/files#diff-c06ef9fa9ce86e5f8888d227fa91d0cfR124

Reply:
Yes, this is due to `FrameworkConfig` requring access to `EnvStore` to pull all the env-vars, and passing in a `EnvStore` wasn't possible at all initialization paths in the scheduler and across the services.

`SchedulerConfig` was easily available however and had easy access to `EnvStore`, as such the context for this is split across these two classes. So its not duplication,even though it looks that way.

The responsibility is split and there is a coupling required here, but so far I haven't found a way to get this done. Happy to hear ways of fixing this as I'm not a fan of this approach either, but havnen't found a better low-impact alternative.